aleo-setup2/Cargo.toml

```diff
@@ -23,20 +23,21 @@ zexe_groth16 = { git = "https://github.com/scipr-lab/zexe", rev = "b24eda5", pac
 zexe_r1cs_core = { git = "https://github.com/scipr-lab/zexe", rev = "b24eda5", package = "r1cs-core", version = "0.1.0" }
 zexe_r1cs_std = { git = "https://github.com/scipr-lab/zexe", rev = "b24eda5", package = "r1cs-std", version = "0.1.0" }
 
+cfg-if = { version = "0.1.10" }
 anyhow = { version = "1.0.31" }
-gumdrop = { version = "0.7.0" }
+gumdrop = { version = "0.7.0", optional = true }
 hex = { version = "0.4.2" }
-hex-literal = { version = "0.1.4" }
-memmap = { version = "0.7.0" }
+hex-literal = { version = "0.1.4", optional = true }
+memmap = { version = "0.7.0", optional = true }
 rand = { version = "0.7.3" }
 rand_xorshift = { version = "0.2.0" }
-rusty-hook = { version = "0.11.2" }
+rusty-hook = { version = "0.11.2", optional = true }
```

Comment:
Can you check, does this need to be optional?

Reply:
yes, rusty-hook doesn't support wasm build